tests/test_ensemble.py

```diff
@@ -1,6 +1,70 @@
 """Test ensemble manipulations"""
-
+import pytest
 from lsstseries import ensemble
+from lsstseries.analysis.stetsonj import calc_stetson_J
+
+
+@pytest.fixture
+def parquet_data():
+    ens = ensemble()
+    ens.from_parquet("tests/data/test_subset.parquet", id_col='ps1_objid', band_col='filterName',
+                     flux_col='psFlux', err_col='psFluxErr')
+
+    return ens
+
+
+def test_from_parquet(parquet_data):
+    """
+    Test that ensemble.from_parquet() successfully loads a parquet file
+    """
+    ens = parquet_data
+
+    assert ens.data is not None  # Check to make sure the data property was actually set
+    ens.data = ens.data.compute()
+    for col in [ens._time_col, ens._flux_col, ens._err_col, ens._band_col]:
+        ens.data[col]  # Check to make sure the critical quantity labels are bound to real columns
+
+
+def test_counts(parquet_data):
+    """
+    Test that ensemble.count() runs and returns the first five values correctly
+    """
+    ens = parquet_data
+
+    count = ens.count()
+    assert list(count.values[0:5]) == [499, 343, 337, 195, 188]
```

Comment:
It's 2000 rows, specifically generated as the first 2000 rows of the ztf_zone dataset Neven put together. 2000 felt sufficient for getting a nice collection of object ids (on order a dozen) that have enough information to produce sensible output in current (and future) analysis functions. Hesitant to add another data file, but it will make sense if/when we support csv, which I do think we'll want at some point.